app/Console/Commands/InitCommand.php

```diff
@@ -121,26 +125,21 @@ private function setUpDatabase(): void
         ]);
     }
 
+    private function inNoInteractionMode(): bool
+    {
+        return (bool) $this->option('no-interaction');
+    }
+
     private function setUpAdminAccount(): void
     {
         $this->info("Let's create the admin account.");
-
-        $name = config('koel.admin.name');
-        if (!$name) {
-            $name = $this->ask('Your name');
+        if ($this->inNoInteractionMode()) {
+            $name     = config('koel.admin.name');
```

Comment:
Please don't align variable assignments.

Reply:
ok, undoing in next commit